src/app/services/baw-api/mock/projectsMockService.ts

```diff
@@ -1,29 +1,39 @@
+import { HttpClient } from "@angular/common/http";
 import { Injectable } from "@angular/core";
-import { Subject } from "rxjs";
+import { Observable } from "rxjs";
 import { Project } from "src/app/models/Project";
-import { MockApiCommon } from "./api-commonMock";
+import { AppConfigService } from "../../app-config/app-config.service";
+import { StandardApi } from "../api-common";
 
 @Injectable({
   providedIn: "root"
 })
-export class MockProjectsService extends MockApiCommon<Project> {
-  public getProject() {
-    return new Subject();
+export class MockProjectsService extends StandardApi<Project, []> {
```

Comment:
I wonder if there is a more general way to create mock services this?

It seems like a lot of work to create a mock service for every class.